tools/layer-zero-example/hardhat.config.js

```diff
@@ -0,0 +1,224 @@
+/*-
+ *
+ * Hedera JSON RPC Relay - Hardhat Example
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+require('dotenv').config();
+require("@nomicfoundation/hardhat-toolbox");
+
+module.exports = {
+  solidity: {
+    compilers: [
+      {
+        version: '0.8.22',
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 200,
+          },
+        },
+      },
+    ],
+  },
+  defaultNetwork: 'hedera_testnet',
+  networks: {
+    hedera_testnet: {
+      url: 'https://testnet.hashio.io/api',
+      accounts: [process.env.HEDERA_PK],
+      chainId: 296,
+    },
+    bsc_testnet: {
+      url: "https://data-seed-prebsc-1-s1.binance.org:8545",
+      chainId: 97,
+      gasPrice: 20000000000,
+      accounts: [process.env.BSC_PK]
+    },
+  }
+};
+
+const getEndpointAddress = (network) => {
+  let ENDPOINT_V2;
+
+  if (network === 'hedera_testnet') {
+    ENDPOINT_V2 = '0xbD672D1562Dd32C23B563C989d8140122483631d';
```

Comment:
Q: is this the official LZ testnet contract or one you pushed for this example.
Either was note this in case the user hits the testnet after a reset.
In which case they'll either need to push one for their tests or point to the LZ one.
You should note this in the README and make sure to link to the endpoint code